pkg/storage/image.go

```diff
@@ -27,6 +28,10 @@ type ImageResult struct {
 	Digests  []string
 	Size     *uint64
 	ImageRef string
+	// TODO(runcom): this is an hack for https://github.com/kubernetes-incubator/cri-o/pull/1136
+	// drop this when we have proper image IDs (as in, image IDs should be just
+	// the config blog digest (which is stable across same images).
```

Comment:
A nit of nits.  Missing end paren, don't worry about it unless you need to touch something else up in here.

Reply:
fixed, thx